apisix/cli/ngx_tpl.lua

```diff
@@ -235,6 +235,10 @@ http {
     lua_shared_dict ext-plugin {* http.lua_shared_dict["ext-plugin"] *}; # cache for ext-plugin
     {% end %}
 
+    {% if config_center == "xds" then %}
+    lua_shared_dict router-config  10m;
```

Comment:
```suggestion
    lua_shared_dict xds-route-config  10m;
```
would that be better?

Reply:
updated